src/backend/web/app.js

```diff
@@ -23,7 +23,7 @@ app.use(
             directives: {
               defaultSrc: ["'self'"],
               fontSrc: ["'self'", 'https:', 'data:'],
-              frameSrc: ["'self'", '*.youtube.com', '*.vimeo.com', '*.twitch.tv'],
+              frameSrc: ["'self'", '*.youtube.com', '*.vimeo.com', 'player.twitch.tv'],
```

Comment:
We don't run this web server anymore right?

Reply:
Nope, can be removed later on